docs/workflow.md

```diff
@@ -133,24 +134,23 @@ When reviewing pull requests, it can be really difficult to phrase comments in a
 
 [https://mtlynch.io/human-code-reviews-2/](https://mtlynch.io/human-code-reviews-2/)
 
+
 ## Example Workflow
 
 We find our workflow is best explained by walking through an example. We're assuming you have already cloned the repository and set up your git remotes. If not, check out our [Getting the Code](software-setup.md#getting-the-code) instructions first and then come back.
 
-This example incorporates information from the previous sections on [Issue and Project Tracking](workflow-and-conventions.md#issue-and-project-tracking), and [the Git Workflow](workflow-and-conventions.md#git-workflow). Make sure you have read those sections first. This example skips over some of the smaller details.
+This example incorporates information from the previous sections on [Issue and Project Tracking](#issue-and-project-tracking), and [the Git Workflow](#git-workflow). Make sure you have read those sections first. This example skips over some of the smaller details.
   
 We are also assuming all the work done here is in your fork of the repository.
 
 Let's pretend our goalie strategy isn't that great. You have noticed that and suggested we improve it. Here's what your workflow would likely look like, from start to finish. We will pretend your name is Bob.
 
 1. Create a new Issue for the goalie strategy if it doesn't already exist
-   1. Since this is more of a new feature than a bug, you would use the "Feature Request" Issue template
    2. Let's pretend this is `Issue #23`
```

Comment:
I think you need to change the numbering here, the rich diff shows these all on one line and with the same level of indenting.

Reply:
It looks ok to me, but I fixed the numbering.